react/ActionMenu/index.jsx

```diff
@@ -1,166 +1,43 @@
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-import Hammer from 'hammerjs'
+import cx from 'classnames'
 import styles from './styles.styl'
-import Overlay from '../Overlay'
-import once from 'lodash/once'
 import { Media, Bd, Img } from '../Media'
+import BottomDrawer from '../BottomDrawer'
+import ClickAwayListener from '@material-ui/core/ClickAwayListener'
+
+const ActionMenuWrapper = ({ inline, onClose, children }) =>
+  inline ? (
+    <ClickAwayListener onClickAway={onClose}>{children}</ClickAwayListener>
```

Comment:
I'm not 100% sure about this. It works as intended, but also makes this component dependent on material-ui. I think it's ok, but we ca achieve the same thing with a transparent `Overlay` component if we'd rather not have the dependency.

Reply:
IMO not a big deal to have a dep to MUI. It's now almost mandatory anyway